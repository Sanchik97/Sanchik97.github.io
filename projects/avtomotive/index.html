<!DOCTYPE html>
<html lang="ru">
	<head>
		<meta charset="UTF-8">
		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
		<title>Title</title>
		<link rel="stylesheet" href="css/animate.min.css">
		<link rel="stylesheet" href="css/normalize.css">
		<link rel="stylesheet" href="css/style.css">
	</head>
	<body class="body">
		<header class="header-section">
			<div class="container">
				<div class="navbar-wrapper">
					<div class="logo"><a class="logo__link" href="#">Avtomotive</a></div>
					<nav class="navbar">
						<ul class="navbar__list">
							<li class="navbar__item"><a href="#" class="navbar__link">Главная</a></li>
							<li class="navbar__item"><a href="#" class="navbar__link">О нас</a></li>
							<li class="navbar__item"><a href="#" class="navbar__link">Услуги</a></li>
							<li class="navbar__item"><a href="#" class="navbar__link">Галерея</a></li>
							<li class="navbar__item"><a href="#" class="navbar__link">Контакты</a></li>
						</ul>
					</nav>
				</div>
				<div class="header-wrapper no-wrap">
					<div class="header-text">
						<h1 class="header-text__title global-title">Продайте вашу <span>машину сейчас</span></h1>
						<p class="header-text__subtitle">Далеко-далеко за словесными горами в стране, гласных и согласных живут рыбные тексты. Своего, текстов маленькая прямо продолжил безорфографичный маленький, которой букв запятых своих ты приставка знаках? Всеми он, проектах бросил. Использовало, алфавит!</p>
					</div>
					<form class="header-form animated bounceInRight">
						<h2 class="header-form__title global-title">Заполните Форму</h2>
						<input class="header-form__input" type="text" placeholder="Введите имя">
						<input class="header-form__input" type="mail" placeholder="E-mail адрес">
						<input class="header-form__input" type="tel" placeholder="Номер телефона">
						<input class="header-form__input" type="text" placeholder="Страна | Город">
						<input class="header-form__checkbox" id="checkbox" type="checkbox">
						<label class="header-form__label" for="checkbox">Я соглашаюсь с требованиями</label>
						<input class="header-form__button" type="button" class="header-form__button" value="Отправить">
					</form>
				</div>
			</div>
		</header>
		<section class="services-section">
			<div class="container">
				<h3 class="services-title global-title"><span>Наши</span> Услуги</h3>
				<div class="services-wrapper">
					<div class="services-block text-center">
						<img class="services-block__img" src="img/services1-min.png" alt="">
						<p class="services-block__text">Далеко-далеко за словесными горами в стране, гласных.</p>
					</div>
					<div class="services-block text-center">
						<img class="services-block__img" src="img/services2-min.png" alt="">
						<p class="services-block__text">Далеко-далеко за словесными горами в стране, гласных.</p>
					</div>
					<div class="services-block text-center">
						<img class="services-block__img" src="img/services3-min.png" alt="">
						<p class="services-block__text">Далеко-далеко за словесными горами в стране, гласных.</p>
					</div>
					<div class="services-block text-center">
						<img class="services-block__img" src="img/services4-min.png" alt="">
						<p class="services-block__text">Далеко-далеко за словесными горами в стране, гласных.</p>
					</div>
				</div>
			</div>
		</section>
		<section class="story-section">
			<div class="container">
				<div class="story-wrapper">
					<p class="story-text">Мы находимся в бизнесе более 6 лет, предоставляя прекрасные услуги клиенту
					и люди любят их до глубины души. Посмотрите нашу историю, чтобы узнать больше.</p>
					<a class="story-button" href="#">Наша история</a>
				</div>
			</div>
		</section>
		<section class="price-section">
			<div class="container">
				<div class="price-text">
					<h3 class="price-text__title global-title">Тарифы</h3>
				</div>
				<div class="price-wrapper">
					<div class="price-block animated text-center">
						<h4 class="price-block__title">Начальный</h4>
						<p class="price-block__price">Free</p>
						<ul class="price-block__list">
							<li class="price-block__item">Бесплатный сервис</li>
							<li class="price-block__item">Несколько аккаунтов</li>
							<li class="price-block__item">Персональный менеджер</li>
							<li class="price-block__item">--</li>
							<li class="price-block__item">--</li>
						</ul>
						<a class="price-block__button" href="#">Попробовать</a>
					</div>
					<div class="price-block text-center">
						<div class="price-block__triangle">
							<p class="price-block__triangle-text">Popular</p>
						</div>
						<h4 class="price-block__title">Бизнесс</h4>
						<p class="price-block__price">$ 97</p>
						<ul class="price-block__list">
							<li class="price-block__item">Бесплатный сервис</li>
							<li class="price-block__item">Несколько аккаунтов</li>
							<li class="price-block__item">Персональный менеджер</li>
							<li class="price-block__item">Безлимитные данные</li>
							<li class="price-block__item">--</li>
						</ul>
						<a class="price-block__button" href="#">Попробовать</a>
					</div>
					<div class="price-block animated text-center">
						<h4 class="price-block__title">Профессиональный</h4>
						<p class="price-block__price">$ 297</p>
						<ul class="price-block__list">
							<li class="price-block__item">Бесплатный сервис</li>
							<li class="price-block__item">Несколько аккаунтов</li>
							<li class="price-block__item">Персональный менеджер</li>
							<li class="price-block__item">Безлимитные данные</li>
							<li class="price-block__item">Все, что вам нужно</li>
						</ul>
						<a class="price-block__button" href="#">Попробовать</a>
					</div>
				</div>
			</div>
		</section>
		<section class="dual-section">
			<div class="container dual-wrapper">
				<div class="reviews-wrapper">
					<h3 class="reviews-title global-title">Отзывы</h3>
					<div class="reviews-block">
						<div class="reviews-block__item reviews-block__item--top">
							<img class="reviews-block__img" src="img/Layer 4-min.png" alt="">
							<p class="reviews-block__text">Я работаю с этой компанией целую вечность и просто хочу сказать, что это отличная работа по купонному дизайну.</p>
							<p class="reviews-block__author">DJ Bravo - Frequent Travelera</p>
						</div>
						<div class="reviews-block__item reviews-block__item--bottom">
							<img class="reviews-block__img" src="img/Layer 6-min.png" alt="">
							<p class="reviews-block__text">Я работаю с этой компанией целую вечность и просто.</p>
							<p class="reviews-block__author">DJ Bravo - Frequent Travelera</p>
						</div>
					</div>
				</div>
				<div class="gallery-wrapper">
					<h3 class="gallery-title global-title">Галерея</h3>
					<div class="gallery-block">
						<div class="gallery-block__row-top d-flex">
							<img src="img/Layer 12-min.png" alt="" class="gallery-block__img">
							<img src="img/Layer 12-min.png" alt="" class="gallery-block__img">
							<img src="img/Layer 12-min.png" alt="" class="gallery-block__img">
						</div>
						<div class="gallery-block__row-bottom d-flex">
							<img src="img/Layer 12-min.png" alt="" class="gallery-block__img">
							<img src="img/Layer 12-min.png" alt="" class="gallery-block__img">
							<img src="img/Layer 12-min.png" alt="" class="gallery-block__img">
						</div>
						<a class="gallery-block__link global-title" href="#">Показать больше картинок</a>
					</div>
				</div>
			</div>
		</section>
		<footer class="footer-section">
			<div class="container">
				<div class="footer-wrapper">
					<div class="copyright">
						<p class="copyright__text">© Copyright | Все права защищены |  2018  </p>
					</div>
					<div class="logo logo--footer"><a class="logo__link logo__link--footer" href="#">Avtomotive</a></div>
				</div>
			</div>
		</footer>
		<script src="js/jquery.min.js"></script>
		<script src="js/main.js"></script>
	</body>
</html>