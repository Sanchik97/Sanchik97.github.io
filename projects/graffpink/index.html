<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
		<title>Title</title>
		<link rel="stylesheet" href="css/animate.min.css">
		<link rel="stylesheet" href="css/bootstrap.min.css">
		<link rel="stylesheet" href="css/owl.carousel.min.css">
		<link rel="stylesheet" href="css/owl.theme.default.min.css">
		<link rel="stylesheet" href="css/style.css">
	</head>
	<body>
		<header class="header-section">
			<div class="navbar-block d-flex align-items-center">
				<div class="container">
					<div class="row">
						<div class="col-lg-3 col-md-4 col-sm-5 col-7">
							<a href="#" class="navbar-block__logo">Graff <span>Pink</span></a>
						</div>
						<div class="col-lg-7 col-md-3 col-sm-3 col-4 ml-auto">
							<nav class="navbar-block__navbar">
								<div class="navbar-block__block">
									<div class="navbar-block__menuToggle text-center">Меню</div>
									<ul class="navbar-block__menu">
										<li class="navbar-block__item"><a class="navbar-block__link" href="#">Home</a></li>
										<li class="navbar-block__item"><a class="navbar-block__link" href="#">About Us</a></li>
										<li class="navbar-block__item"><a class="navbar-block__link" href="#">Services</a></li>
										<li class="navbar-block__item"><a class="navbar-block__link" href="#">Contacts</a></li>
										<li class="navbar-block__item"><a class="navbar-block__link navbar-block__button" href="#"><img class="navbar-block__img img-fluid" src="img/Shape 1-min.png" width="13px" height="14px" alt="">Login</a>
										
										<div class="navbar-block__dropdown text-center">
											<span class="navbar-block__title"></span>
											<hr>
											<form action="">
												<input class="navbar-block__input" type="mail" placeholder="Enter your E-mail">
												<input class="navbar-block__input" type="password" placeholder="Enter your password">
												<input class="navbar-block__input navbar-block__button" type="button" value="Submit">
											</form>
										</div>
									</li>
								</ul>
							</div>
						</nav>
					</div>
				</div>
			</div>
		</div>
		<div class="header-block text-center">
			<h1 class="header-block__title">Welcome to Pink</h1>
			<p class="header-block__subtitle">Lorem ipsum dolor sit amet, consectetur.</p>
		</div>
	</header>
	<section class="features-section">
		<div class="container features">
			<div class="features-text text-center">
				<h2 class="features-text__title">
				Lorem ipsum is simply dummy
				</h2>
				<p class="features-text__subtitle m-auto">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown</p>
			</div>
			<div class="features-block d-flex flex-nowrap justify-content-around">
				<div style="background-color:#4EB9F4;" class="features-block__item d-flex flex-column align-items-center justify-content-around">
					<img class="features-block__img img-fluid" src="img/tickets-min.png" width="80px" height="82px" alt="tickets">
					<p class="features-block__title">Tickets</p>
				</div>
				<div style="background-color:#6FC125;" class="features-block__item d-flex flex-column align-items-center justify-content-around">
					<img class="features-block__img img-fluid" src="img/camera-min.png" width="80px" height="82px" alt="tickets">
					<p class="features-block__title">Camera</p>
				</div>
				<div style="background-color:#FF5649;" class="features-block__item d-flex flex-column align-items-center justify-content-around">
					<img class="features-block__img img-fluid" src="img/beermug-(Cheers_)-min.png" width="80px" height="82px" alt="tickets">
					<p class="features-block__title">Beer</p>
				</div>
				<div style="background-color:#E48F23;" class="features-block__item d-flex flex-column align-items-center justify-content-around">
					<img class="features-block__img img-fluid" src="img/iPad-potrait-&-landscape-2-min.png" width="80px" height="82px" alt="tickets">
					<p class="features-block__title">Devices</p>
				</div>
			</div>
		</div>
	</section>
	<section class="content-section">
		<div class="container">
			<div class="content-block">
				<div class="content-block__item">
					<img class="content-block__img img-fluid" src="img/12-outdoor-planters-220x150.jpg" width="220" height="150" alt="">
					<h3 class="content-block__title">Lorem ipsum is simply</h3>
					<p class="content-block__subtitle">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. </p>
				</div>
				<div class="content-block__item">
					<img class="content-block__img img-fluid" src="img/17-outdoor-planters-220x150.jpg" width="220" height="150" alt="">
					<h3 class="content-block__title">Lorem ipsum is simply</h3>
					<p class="content-block__subtitle">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. </p>
				</div>
				<div class="content-block__item">
					<img class="content-block__img img-fluid" src="img/27-outdoor-planters-220x150.jpg" width="220" height="150"  alt="">
					<h3 class="content-block__title">Lorem ipsum is simply</h3>
					<p class="content-block__subtitle">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. </p>
				</div>
			</div>
		</div>
	</section>
	<footer class="footer-section">
		<div class="container">
			<div class="row">
				<div class="col-lg-8 col-md-12 col-sm-12 flex-nowrap col-12 d-flex">
					<div class="footer-block">
						<h4 class="footer-block__title">Sub Menu</h4>
						<ul class="footer-block__list">
							<li class="footer-block__item"><a class="footer-block__link" href="#">Android</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Apps</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Articles</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Backgrounds</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Blogger</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Books</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Coding</a></li>
						</ul>
					</div>
					<div class="footer-block">
						<h4 class="footer-block__title">Sub Menu</h4>
						<ul class="footer-block__list">
							<li class="footer-block__item"><a class="footer-block__link" href="#">Android</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Apps</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Articles</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Backgrounds</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Blogger</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Books</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Coding</a></li>
						</ul>
					</div>
					<div class="footer-block">
						<h4 class="footer-block__title">Sub Menu</h4>
						<ul class="footer-block__list">
							<li class="footer-block__item"><a class="footer-block__link" href="#">Android</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Apps</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Articles</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Backgrounds</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Blogger</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Books</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Coding</a></li>
						</ul>
					</div>
					<div class="footer-block">
						<h4 class="footer-block__title">Sub Menu</h4>
						<ul class="footer-block__list">
							<li class="footer-block__item"><a class="footer-block__link" href="#">Android</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Apps</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Articles</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Backgrounds</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Blogger</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Books</a></li>
							<li class="footer-block__item"><a class="footer-block__link" href="#">Coding</a></li>
						</ul>
					</div>
				</div>
				
					<div class="col-lg-4 col-md-12 footer-block footer-copyright">
						<ul class="d-flex list-inline">
							<li>
								<a href="#">
									<div class="footer-block__sprite facebook"></div>
								</a>
							</li>
							<li>
								<a href="#">
									<div class="footer-block__sprite twitter"></div>
								</a>
							</li>
							<li>
								<a href="#">
									<div class="footer-block__sprite dribble"></div>
								</a>
							</li>
							<li>
								<a href="#">
									<div class="footer-block__sprite share"></div>
								</a>
							</li>
						</ul>
						<p class="footer-block__copyright">Copyright © 2009–2018 Sanchkk</p>
					</div>
			</div>
		</div>
	</footer>
	<script src="js/jquery.min.js"></script>
	<script src="js/owl.carousel.min.js"></script>
	<script src="js/bootstrap.min.js"></script>
	<script src="js/main.js"></script>
</body>
</html>