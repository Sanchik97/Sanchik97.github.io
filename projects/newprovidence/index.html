<!DOCTYPE html>
<html lang="ru">
	<head>
		<meta charset="UTF-8">
		<title>Title</title>
		<link rel="stylesheet" href="css/bootstrap.min.css">
		<link rel="stylesheet" href="css/style.css">
	</head>
	<body>
		<header class="header">
			<div class="container">
				<div class="row navbar d-flex align-items-around">
					<div class="col-lg-5 col-sm-4">
						<nav>
							<div class="wrapper d-flex align-items-center">
								<div class="menuToggle"><img src="img/menu.png" alt="menu"></div>
								<ul class="menu">
									<li class="menu__item"><a class="menu__link" href="#">Tour</a></li>
									<li class="menu__item"><a class="menu__link" href="#">Features</a></li>
									<li class="menu__item"><a class="menu__link" href="#">Pricing</a></li>
								</ul>
							</div>
						</nav>
					</div>
					<div class="col-lg-3 col-sm-3 d-flex align-items-center logo"><a class="logo__link" href="#">newprovidence</a></div>
					<div class="col-lg-4 col-sm-4 d-flex align-items-center justify-content-end">
						<ul class="menu">
							<li class="menu__item">
								<a href="#" class="menu__link">Help</a>
								<a href="#" class="menu__link">Contacts</a>
								<a href="#" class="menu__link button"><img class="button__img" src="img/appleicon.png" alt="">Get App</a>
							</li>
						</ul>
					</div>
				</div>
				<div class="heading m-auto text-center">
					<h1 class="heading__title ">What happens tomorrow?</h1>
					<p class="heading__subtitle">The sight of the tumblers restored Bob Sawyer to a degree of equanimity which he had not possessed since his interview with his landlady. His face brightened up, and he began to feel quite convivial.</p>
					<a class="button heading__button" href="#"><img class="heading__img" src="img/playicon.png" alt="">Watch Video</a>
				</div>
				<div class="phone text-center">
					<img src="img/phone1.png" alt="">
				</div>
			</div>
		</header>
		<section>
			<div class="container">
				<div class="row d-flex flex-nowrap justify-content-around">
					<div class="col-lg-6 features">
						<img class="features__img" src="img/security.png" alt="">
						<p class="features__title">Real Time all the time</p>
						<p class="features__subtitle">
							Thus much I thought proper to tell you in relation to yourself, and to the trust I reposed in you.
						</p>
						<p>
							Just then her head struck against the roof of the hall: in fact she was now more than nine feet high, and she at once took up the little golden key and hurried off to the garden door.
						</p>
					</div>
					<div class="col-lg-6">
						<img class="features__img" src="img/clock.png" alt="">
						<p class="features__title">Adopt without any obstacles</p>
						<p class="features__subtitle">
							This sounded a very good reason, and Alice was quite pleased to know it. 'I never thought of that before!' she said.
						</p>
						<p>
							This I have produced as a scantling of Jackâ€™s great eloquence and the force of his reasoning upon such abstruse matters.
						</p>
					</div>
				</div>
				<hr class="features__line">
			</div>
		</section>
		<section class="protected">
			<div class="container">
				<div class="row">
					<div class="col-lg-5 protected-block ml-auto">
						<h2 class="protected-block__title">Your day is protected</h2>
						<p class="protected-block__subtitle">There have not been any since we have lived here, said my mother.</p>
						<p class="protected-block__subtitle">We thought - Mr. Copperfield thought - it was quite a large rookery; but the nests were very old ones, and the birds have deserted them a long while.</p>
						<a class="button protected-block__button1" href="#">Try to hack us <img src="img/arrowicon.png" alt=""></a><a class="button protected-block__button2" href="#">Learn More</a>
					</div>
				</div>
			</div>
		</section>
		<section class="worldmap">
			<div class="container">
				<div class="row">
					<div class="col-lg-5 worldmap-block">
						<h2 class="worldmap-block__title">We are almost everywhere</h2>
						<p class="worldmap-block__subtitle">Now the races of these two have been for some ages utterly extinct, and besides to discourse any further of them would not be at all to my purpose.</p>
						<p class="worldmap-block__subtitle">Thus much I thought proper to tell you in relation to yourself, and to the trust I reposed in you.</p>
						<p class="worldmap-block__partners">partners</p>
						<div class="partners">
							<ul class="partners-sprite d-flex ">
								<li>
									<a href="#">
										<div class="partners-sprite dropbox"></div>
									</a>
								</li>
								<li>
									<a href="#">
										<div class="partners-sprite evernote"></div>
									</a>
								</li>
							</ul>
						</div>
					</div>
				</div>
			</div>
		</section>
		<section class="interface text-center">
			<h2 class="interface__title">Beautifull Interface</h2>
			<hr class="interface__line m-auto"/>
			<p class="interface__text">landing screen</p>
			<img class="img-fluid w-100" src="img/Screens.png" alt="">
			<p class="interface__text"><b>new</b>providence</p>
			<h2 class="interface__title">Get it to your smartphone</h2>
			<form action="" method="_GET">
				<input placeholder="Enter your E-mail" class="form__input" type="mail"><input class="form__button " type="button" value="Send invite"><img class="form__img" src="img/arrowicon.png" alt="">
			</form>
			<p class="interface__text">Or download from</p>
			<a href="https://itunes.apple.com/us/app/amazon-shopping-made-easy/id297606951?mt=8" style="display:inline-block;overflow:hidden;background:url(https://linkmaker.itunes.apple.com/assets/shared/badges/en-us/appstore-lrg.svg) no-repeat;width:135px;height:40px;background-size:contain; border-radius: 25px;margin: 10px 0 135px 0; "></a>
		</section>
		
		<section class="perfect-plan">
			<div class="container">
				<div class="row">
					<div class="col-lg-4 col-sm-12 plan">
						<h2 class="plan__title">Choose your perfect plan</h2>
						<span class="plan__text">you are</span>
						<p class="plan__subtitle">Thus much I thought proper to tell you in relation to yourself, and to the trust I reposed in you.</p>
						<p>Have a bigger team? <a class="plan__link" href="#">Let's Talk</a></p>
					</div>
					<div class="d-flex plan-mobile flex-nowrap col-lg-8 col-sm-12">
						<div class="plan-block plan-block--left ml-auto text-center">
							<h2 class="plan-block__title">Starter</h2>
							<a href="#" class="plan-block__button">FREE</a>
							<p class="plan-block__subtitle">Build your schedule <br>every day</p>
							<ul class="plan-block__list">
								<li class="plan-block__items">Unlimited events</li>
								<li class="plan-block__items">Connect Dropbox & Evernote</li>
							</ul>
							<a class="button plan-block__button2" href="#">Get started</a>
						</div>
						<div class="plan-block plan-block--right  text-center">
							<h2 class="plan-block__title">PRO</h2>
							<a href="#" class="plan-block__button">$4.99</a>
							<p class="plan-block__subtitle">Make your live <br>better</p>
							<ul class="plan-block__list">
								<li class="plan-block__items">Unlimited events</li>
								<li class="plan-block__items">Connect Dropbox & Evernote</li>
								<li class="plan-block__items">Personal Assistant</li>
							</ul>
							<a class="button plan-block__button2" href="#">Get started</a>
						</div>
					</div>
				</div>
			</div>
		</section>
		<section>
			<div class="container text-center">
				<p class="interface__text">newprovidence</p>
				<h2>Get it to your smartphone</h2>
				<form action="" method="_GET">
					<input placeholder="Enter your E-mail" class="form__input" type="mail"><input class="form__button " type="button" value="Send invite"><img class="form__img" src="img/arrowicon.png" alt="">
				</form>
				<p class="interface__text">We promise to never spam you</p>
				<hr class="features__line">
			</div>
		</section>
		<footer class="footer">
			<div class="container">
				<div class="row">
					<div class="col-lg-3">
						<div class="col-lg-2 logo"><a class="logo__link" href="#">newprovidence</a></div>
						<p class="footer__text">It was some time before he obtained any answer, and the reply, when made, was unpropitious.</p>
						<a href="https://itunes.apple.com/us/app/amazon-shopping-made-easy/id297606951?mt=8" style="display:inline-block;overflow:hidden;background:url(https://linkmaker.itunes.apple.com/assets/shared/badges/en-us/appstore-lrg.svg) no-repeat;width:135px;height:40px;background-size:contain; margin: 35px 0 35px 15px; "></a>
						<div class="social">
							<ul class="d-flex">
								<li>
									<a href="#">
										<div class="social-sprite twitter"></div>
									</a>
								</li>
								<li>
									<a href="#">
										<div class="social-sprite facebook"></div>
									</a>
								</li>
								<li>
									<a href="#">
										<div class="social-sprite google"></div>
									</a>
								</li>
								<li>
									<a href="#">
										<div class="social-sprite tumbler"></div>
									</a>
								</li>
								<li>
									<a href="#">
										<div class="social-sprite pinterest"></div>
									</a>
								</li>
								<li>
									<a href="#">
										<div class="social-sprite linkedin"></div>
									</a>
								</li>
							</ul>
						</div>
					</div>
					<div class="col-lg-6 d-flex justify-content-around ml-auto">
						<ul class="footermenu">
							<li class="footermenu__list"><h3 class="footermenu__title">Learn More</h3></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">How it works?</a></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">Meeting Tools</a></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">Live striming</a></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">Contact method</a></li>
						</ul>
						<ul class="footermenu">
							<li class="footermenu__list"><h3 class="footermenu__title">About Us</h3></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">About Us</a></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">Features</a></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">Privacy policy</a></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">Terms & Conditions</a></li>
						</ul>
						<ul class="footermenu">
							<li class="footermenu__list"><h3 class="footermenu__title">Support</h3></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">F.A.Q</a></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">Contact Us</a></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">Live chat</a></li>
							<li class="footermenu__list"><a class="footermenu__link" href="">Phone call</a></li>
						</ul>
					</div>
				</div>
				<hr class="features__line">
				<div class="copyright d-flex align-items-center justify-content-center">
					<p class="copyright__text">copyrightÂ© 2018</p>
				</div>
			</div>
		</footer>
		<script src="js/jquery.min.js"></script>
		<script src="js/bootstrap.min.js"></script>
		<script src="js/main.js"></script>
	</body>
</html>