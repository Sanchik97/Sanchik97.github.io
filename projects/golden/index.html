<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<title>Golden - studio</title>
		<link rel="stylesheet" href="css/style.css"/>
	</head>
	<body>
		<header>
			<div class="container">
				<div class="clearfix">
					<img src="img/logo.png" alt="Golden" class="logo">
					<nav>
						<ul class="navbar">
							<li>
								<a href="#">home</a>
								<a href="#services">services</a>
								<a href="#portfolio">portfolio</a>
								<a href="#about">about</a>
								<a href="#contact">contact</a>
							</li>
						</ul>
					</nav>
				</div>
				<p class="title_first">welcome to our studio!</p>
				<h1>it's nice to meet you!</h1>
				<a href="#" class="button">tell me more</a>
				
			</div>
		</header>
		<section id="services">
			<div class="container">
				<div class="title">
					<h2>services</h2>
					<p>Proin iaculis purus consequat sem cure.</p>
				</div>
				<div class="clearfix">
					<div class="section">
						<img src="img/icon1.png" alt="">
						<h3>E-Commerce</h3>
						<p>Proin iaculis purus consequat sem cure
							digni ssim. Donec porttitora entum suscipit
						aenean rhoncus posuere odio in tincidunt.</p>
					</div>
					<div class="section">
						<img src="img/icon2.png" alt="">
						<h3>Responsive Web</h3>
						<p>Proin iaculis purus consequat sem cure
							digni ssim. Donec porttitora entum suscipit
						aenean rhoncus posuere odio in tincidunt.</p>
					</div>
					<div class="section">
						<img src="img/icon3.png" alt="">
						<h3>Web Security</h3>
						<p>Proin iaculis purus consequat sem cure
							digni ssim. Donec porttitora entum suscipit
						aenean rhoncus posuere odio in tincidunt.</p>
					</div>
				</div>
			</div>
		</section>
		<section id="portfolio">
			<div class="container">
				<div class="title">
					<h2>portfolio</h2>
					<p>Proin iaculis purus consequat sem cure.</p>
				</div>
				<div class="image">
					<a href="#"><img src="img/img1.jpg" alt=""></a>
					<a href="#"><img src="img/img2.jpg" alt=""></a>
					<a href="#"><img src="img/img3.jpg" alt=""></a>
				</div>
			</div>
		</section>
		<section id="about">
			<div class="container">
				<div class="title">
					<h2>about us</h2>
					<p>Proin iaculis purus consequat sem cure.</p>
				</div>
				<table>
					<tr>
						<td><div class="leftside">
							<img class="leftimage" src="img/about1.jpg" alt="">
							<h4><span>July 2010</span><br/>Our Humble Beginnings<h4>
							<br/>
							<p>Proin iaculis purus consequat sem cure
								digni ssim. Donec porttitora entum suscipit
								aenean rhoncus posuere odio in tincidunt. Proin
								iaculis purus consequat sem cure digni
							ssim. Donec porttitora entum suscipit.</p>
						</div>
					</td>
					<td></td>
				</tr>
				<tr>
					<td>
						
					</td>
					<td>
						<div class="rightside">
							<img class="rightimage" src="img/about2.jpg" alt="">
							<h4><span>January 2011</span><br/>Our Humble Beginnings<h4>
							<br/>
							<p>Proin iaculis purus consequat sem cure
								digni ssim. Donec porttitora entum suscipit
								aenean rhoncus posuere odio in tincidunt. Proin
								iaculis purus consequat sem cure digni
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						
					</td>
				</tr>
				<tr>
						<td><div class="leftside">
							<img class="leftimage" src="img/about3.jpg" alt="">
							<h4><span>December 2010</span><br/>Our Humble Beginnings<h4>
							<br/>
							<p>Proin iaculis purus consequat sem cure
								digni ssim. Donec porttitora entum suscipit
								aenean rhoncus posuere odio in tincidunt. Proin
								iaculis purus consequat sem cure digni
							ssim. Donec porttitora entum suscipit.</p>
						</div>
					</td>
					<td></td>
				</tr>
				<tr>
					<td>
						
					</td>
					<td>
						<div class="rightside">
							<img class="rightimage" src="img/about4.jpg" alt="">
							<h4><span>April 2012</span><br/>Our Humble Beginnings<h4>
							<br/>
							<p>Proin iaculis purus consequat sem cure
								digni ssim. Donec porttitora entum suscipit
								aenean rhoncus posuere odio in tincidunt. Proin
								iaculis purus consequat sem cure digni
							ssim. Donec porttitora entum suscipit.</p>
						</div>
						
					</td>
				</tr>
			</table>
			
		</div>
	</section>
	<footer>
		<div class="container">
			<p class="copyright">Copyright 2014 FreebiesXpress.com</p>
		</div>
		
	</footer>
</body>
</html>